### DISCUSSION TITLE
Mech Welders and Vanilla Welders.

### DISCUSSION THREAD

**Discussion by WhatColor82** [PeasantFoundry/Mechtrauma#77] [Ideas]
I propose we attempt to differentiate mech welders from vanilla welders. I think we should make it so that it has its own argon bottle for example. Only mech welders can take advantage of argon station while regular welders have to use ethanol.
I feel like regular welding tanks filling up at argon station is actually a bit OP and makes ethanol an afterthought. Or we could also make it scale with mechanical skill and make it stronger than a vanilla welder so that there is reason to use it over vanilla with the added risk. I added several mech welders to my sub but the instant the campaign started they were all scrapped for vanillas! People need a reason to use them! People are at least using the argon machine but I feel the argon machine is a bit too efficient! Hence my idea for making it so that only the mech welders can take advantage which would also bring back ethanol in usefulness.

**Comment by IgRAzm** [comment]
I enjoy that you brought attention to the existing issue of welders, not sure tho that separating welding fuel into types is viable solution. It'd need its own item and overworld sprite, and fuel fitting into one welder but not another is logistical pain for a crew that didn't make a unified choice to use one welder above another.

**Comment by MrBandoIruka** [comment]
Maybe the MT welders can consume welding filler rods as well as fuel to perform said weld

**Reply by WhatColor82** [reply]
I like this!

**Reply by PeasantFoundry** [reply]
The welder situation is the result of an uneasy truce from long ago and its probably time to sort it out once and for all. The mod has tried to override as little vanilla content as possible and the mechtrauma welders were rather difficult to implement in XML only. As the mod has incorporated more lua functionality, these changes are fare more realistic.

**Reply by MapleWheels** [reply]
Good idea. We could probably make a dual-fuel welder that's much faster than the vanilla welder in exchange, while nerfing the vanilla welding speed at the same time.
